src/main/java/edu/unc/lib/boxc/migration/cdm/status/SourceFilesSummaryService.java

```diff
@@ -20,64 +28,57 @@
  */
 public class SourceFilesSummaryService extends AbstractStatusService {
     private static final Logger log = getLogger(SourceFilesSummaryService.class);
+    private static final String ID_FIELD = "id";
+    private static final String MATCHING_VALUE_FIELD = "matching_value";
+    private static final String SOURCE_FILE_FIELD = "source_file";
+    private static final String POTENTIAL_MATCHES_FIELD = "potential_matches";
```

Comment:
We should use the constants in SourceFilesInfo instead of local constants, so that they are more likely to stay in synch